### PR TITLE
create navbar component

### PR DESCRIPTION
**Pull request by gdams**
moves all nav-bar related html/css into a single component

### CODE REVIEW

File: src/components/navbar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  } else {
+    header = (
+      <Link to={"/"} style={{ boxShadow: "none", color: "inherit", lineHeight: "0px" }}>
+        <img className="logo" alt="AdoptOpenJDK logo" src={withPrefix("adopt_logo_white.svg")} style={logoStyle} />
+      </Link>
+    );
+  }
+
+  return (
+    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", background: "url(https://adoptopenjdk.net/dist/assets/background.png)", justifyContent: "center", alignItems: "center", height: "6em"}}>
```
Comment (aahlenst) [inline src/components/navbar.js:29]: Suggestion, takes the local file (reduces risk of being impacted by changes made to the website):

```diff
diff --git a/src/components/navbar.js b/src/components/navbar.js
index 5341c7a..b23f35c 100644
--- a/src/components/navbar.js
+++ b/src/components/navbar.js
@@ -1,6 +1,7 @@
 import React from "react";
 import { Link, withPrefix } from "gatsby";
 
+import Background from "../images/background.png";
 import SocialBar from "./socialbar";
 
 const NavBar = (location) => {
@@ -26,7 +27,7 @@ const NavBar = (location) => {
   }
 
   return (
-    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", background: "url(https://adoptopenjdk.net/dist/assets/background.png)", justifyContent: "center", alignItems: "center", height: "6em"}}>
+    <header id="site-header-main" style={{ display: "flex", backgroundColor: "#152935", backgroundImage: `url(${Background})`, justifyContent: "center", alignItems: "center", height: "6em"}}>
       {header}
       <SocialBar />
     </header>
```

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "gatsby-source-filesystem": "2.3.1",
     "gatsby-transformer-sharp": "2.5.0",
     "prismjs": "1.20.0",
+    "radium": "0.26.0",
```
[the remark this answers is not in the document]
Comment (gdams) [reply package.json:35]: I can investigate changing this but let's use it for now as it works

File: src/components/navbar.js (hunk 1 above)
Comment (gdams) [reply src/components/navbar.js]: changed

**Comment by gdams** [comment]
@aahlenst updated PTAL

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gdams) [reply package.json]: updated, switched to Glamor

**Pull request merged**